docs/taskruns.md

```diff
@@ -402,6 +402,16 @@ case is when your CI system autogenerates `TaskRuns` and it has `Parameters` it
 provide to all `TaskRuns`. Because you can pass in extra `Parameters`, you don't have to
 go through the complexity of checking each `Task` and providing only the required params.
 
+#### Parameter Enums
```

Comment:
are we going to the inline spec case? I think it was discussed before but I forgot the decision

Reply:
Yeah, it is allowed for compatibility reason. But I'm not sure if we want to put it to the documentation, since it is a very edge case and we don't know if there is any usecase for it.

Reply:
Maybe we can mention that this is a valid case but there is no meaning in using it. Since we won't explicitly disallow it. I think just documenting the behaviour might help clear some confusions.

Reply:
Doc updated, PTAL!